google-cloud-clients/google-cloud-spanner/pom.xml

```diff
@@ -18,6 +18,7 @@
   </parent>
   <properties>
     <site.installationModule>google-cloud-spanner</site.installationModule>
+    <skipUnitTests>${skipTests}</skipUnitTests>
```

Comment:
lets not skip tests during install. it is helpful sanity check. u can use -DskipTests if tests are not required

Reply:
This does not skip tests during install. It only sets the default value of skipUnitTests to skipTests.